server/routes/pubDocument.tsx

```diff
@@ -235,3 +251,42 @@ app.get(
 		}
 	},
 );
+
+app.get(['/pub/:pubSlug/review/:historyKey/'], async (req, res, next) => {
```

Comment:
Perhaps this be merged into the Pub draft handler above in the following way:

1. Adding `'/pub/:pubSlug/review/:historyKey/' to the array of routes it handles
2. Checking `req.path` to see whether we're handing the Pub in review mode

I encourage you to give this some thought and then decide for yourself whether this is a good idea. It could also be premature to merge these two handlers, since their behavior may diverge sharply in the future! In this case, since they are very nearly identical and there is a lot of tricky logic that has to be duplicated between them, I would probably merge them — or at least factor some of the shared `historyKey` munging into a helper fn.